### DISCUSSION TITLE
Question on chapter 7 - Neutralizing / Carbon Removal

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/discussions#105] [Community Working Group]
We have received the following feedback from a person who enrolled on the LFC131 course:

@greenhsu123 what are your thoughts?

---

What I don’t understand: In chapter 7 the info-box about Neutralizing / Carbon Removal says:
Solutions that rely on Earth’s natural carbon cycle have short-term durability measured in decades. For example, forestry projects have a durability of 40 to 100 years.

My understanding is that for forestry projects it very much depends on how the forest is managed. E.g. when trees are transformed into building material, the carbon stays embedded in whatever is built with it. Trees that are left to rot will partly release their carbon, but partly be transformed into soil.

It might be that I’m wrong about that, do you have more information about that for me?

I also think the section is slightly too favorable of direct air capture considering that there are currently little projects (I know of) that do DAC in large scales. But again, I’m no expert on the subject matter and would be happy to learn more :slightly_smiling_face:

Since the whole section is also embedded in a context that makes clear that neutralisation is a last resort and not the preferable solution for lowering emissions, I don’t think these are major problems anyways, and I think the training contents overall are very good, thanks for offering this course!

**Comment by NAMRATA-WOKE** [comment]
@russelltrow was this question ever answered in any other space?

**Reply by russelltrow** [reply]
Not that I am aware of.
